src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -548,3 +548,52 @@ def load_nicknames_data():
     nicknames = nicknames.apply(lambda x: x.astype(str).str.title()).set_index("name")
     nicknames = nicknames.replace("Nan", np.nan)
     return nicknames
+
+
+def extract_metadata_proportions(final_output_dir: Path) -> None:
+    # Get directories for each dataset which will contain all shard metadata files
+    dataset_metadata_dfs = []
+    directories = [
```

Comment:
Can we explicitly iterate over dataset names here? That seems clearer and would continue to work if we added any more directories in addition to "logs".

Reply:
I think that should work.